src/Exception/UnavailableActionException.php

```diff
@@ -4,9 +4,20 @@
 
 use Throwable;
 
+/**
+ * Class UnavailableActionException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle\Exception
+ */
 class UnavailableActionException extends \Exception
 {
-    public function __construct($message = "Action unavailable", $code = 0, Throwable $previous = null)
+    /**
+     * UnavailableActionException constructor.
+     * @param string $message
+     * @param int $code
+     * @param Throwable|null $previous
+     */
+    public function __construct($message = "Unavailable Acition", $code = 0, Throwable $previous = null)
```

Comment:
is this part of this task?

Reply:
Traits need Exceptions so IMO yes
setMethod need translate addable, getable ect to POST GET and if fail we need exception

Reply:
but what are you changing here? :smile: 

missing docblock and default param? :smile: